src/Diagnostics.RuntimeHost/Services/DevOpsClient/DevOpsClient.cs

```diff
@@ -0,0 +1,237 @@
+﻿using Diagnostics.RuntimeHost.Utilities;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Diagnostics.RuntimeHost.Services.DevOpsClient
+{
+    public interface IDevOpsClient : IDisposable, IRepoClient
+    {
+        string AccessToken { get; }
+        string Organization { get; }
+        string RepositoryID { get; }
+        string Project { get; }
+    }
+
+    public class DevOpsResponse
+    {
+        public object result;
+        public HttpStatusCode responseCode;
+    }
+
+    class DevOpsClient : IDevOpsClient
```

Comment:
I see that `IRepoClient` is the one with the methods defined. Shouldn't that be the interface here?